supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorWrapper.scala

```diff
@@ -68,13 +68,21 @@ echo $$? > rc
     ()
   }
 
-  def generateSubmitFile(path: Path, attributes: Map[String, Any]): String = {
+  def generateSubmitFile(path: Path, attributes: Map[String, Any], nativeSpecs: Option[Array[String]]): String = {
     def htCondorSubmitCommand(filePath: Path) = {
       s"${HtCondorCommands.Submit} ${filePath.toString}"
     }
 
     val submitFileWriter = path.untailed
     attributes.foreach(attribute => submitFileWriter.writeWithNewline(s"${attribute._1}=${attribute._2}"))
+    //Native specs is intended for attaching HtCondor native configuration such as 'requirements' and 'rank' definition
+    //directly to the submit file.
+    nativeSpecs match {
```

Comment:
Since the None case doesn't do anything, maybe this is shorter and sweeter:
```scala 
nativeSpecs.foreach(_.foreach(submitFileWriter.writeWithNewline))
```